src/makefile.mk

```diff
@@ -26,7 +26,7 @@ BSSHEAP_LOW ?= D052C6
 BSSHEAP_HIGH ?= D13FD8
 STACK_HIGH ?= D1A87E
 INIT_LOC ?= D1A87F
-OUTPUT_MAP ?= NO
+OUTPUT_MAP ?= YES
```

Comment:
was that done on purpose here

Reply:
I disagree with this modification

Reply:
Yes, this was done on purpose. I don't see why anyone wouldn't want this extra build artifact that makes debugging in CEmu not a total nightmare.

Mateo mentioned something about it being slow on Windows, but I tried building oiram with and without this and saw no noticeable change in build time.

Reply:
Seems reasonable; I guess things have improved a lot